linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_minlen_login_defs/rule.yml

```diff
@@ -47,6 +47,7 @@ references:
     nist-csf: PR.AC-1,PR.AC-6,PR.AC-7
     srg: SRG-OS-000078-GPOS-00046
     stigid@ol8: OL08-00-020231
+    stigid@rhel8: RHEL-08-020231
```

Comment:
I couldn't find the `accounts_password_minlen_login_defs` rule or the `RHEL-08-020231` reference in the STIG profile file. Could you confirm the change, please?